packages/manager/modules/overthebox/src/overthebox/actions/overTheBox-actions.controller.js

```diff
@@ -0,0 +1,63 @@
+export default class OverTheBoxActionsCtrl {
+  /* @ngInject */
+
+  constructor($translate, $q, PAGINATION_PER_PAGE, OvhApiOverTheBoxDevice) {
+    this.$translate = $translate;
+    this.$q = $q;
+    this.PAGINATION_PER_PAGE = PAGINATION_PER_PAGE;
+    this.OvhApiOverTheBoxDevice = OvhApiOverTheBoxDevice;
+  }
+
+  $onInit() {
+    this.loaders = {
+      init: true,
+    };
+
+    this.actionIds = [];
+    this.filter = {
+      perPage: this.PAGINATION_PER_PAGE,
+    };
+    this.isError = false;
+
+    this.$q.all([this.getActions()]);
```

Comment:
```suggestion
    return this.getActions();
```
Which implies to add a return at L28